_includes/components/department-budgets/ExpenditureChart/index.jsx

```diff
@@ -109,11 +52,7 @@ export default function ExpenditureChart(props) {
             </div>
           </div>
           <div className="u-marginLeft30">
-            {dataset && buildDataset()}
-            {guide && buildGuide()}
-            {pdf && buildPdf()}
-            {excel && buildExcel()}
-            {cpi && buildCpi()}
+            <Linkslist listArray={linksListArray} />
```

Comment:
For the other charts, we do this all server-side (in Jekyll). Can we not do the same here?

Reply:
@longhotsummer This is actually an oversight on my side. `ExpenditureChart` is actually redundant now and can be removed. Will do so now.